crates/sui-indexer-alt-jsonrpc/src/context.rs

```diff
@@ -64,11 +64,17 @@ impl Context {
             PgReader::new(database_url, db_args, metrics.clone(), registry, cancel).await?;
         let pg_loader = Arc::new(pg_reader.as_data_loader());
 
-        let kv_loader = if let Some(config) = config.bigtable.clone() {
-            let bigtable_reader = BigtableReader::new(config.instance_id, registry).await?;
-            KvLoader::new_with_bigtable(Arc::new(bigtable_reader.as_data_loader()))
+        let config_arc = Arc::new(config.clone());
```

Comment:
As per the comment in the parent PR -- `Context` should be passed the duration directly,  and can then pass it on to `pg_reader`, and if necessary, `bigtable_reader`, below.